cloudflare/resource_cloudflare_worker_kv_test.go

```diff
@@ -0,0 +1,91 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccCloudflareWorkersKV_Basic(t *testing.T) {
+	t.Parallel()
+	var kvPair cloudflare.WorkersKVPair
+	name := generateRandomResourceName()
+	key := generateRandomResourceName()
+	value := generateRandomResourceName()
+	resourceName := "cloudflare_workers_kv." + name
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
If we're testing account level functionality, you'll also want to use [`testAccPreCheckAccount`](https://github.com/terraform-providers/terraform-provider-cloudflare/blob/57ef25dc04a6605d8ec8b4088f1a62f507be5f63/cloudflare/provider_test.go#L62-L66) to ensure the account ID is also defined.